self-tests/terminal.robot

```diff
@@ -0,0 +1,44 @@
+*** Settings ***
+Documentation       This suite verifies the correct operation of keywords
+...                 entering and parsing UEFI shell commands
+
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=30 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../pikvm-rest-api/pikvm_comm.robot
+
+# TODO:
+# - document which setup/teardown keywords to use and what are they doing
+# - go threough them and make sure they are doing what the name suggest (not
+# exactly the case right now)
+Suite Setup         Run Keyword
+...                     Prepare Test Suite
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+Execute UEFI Shell Command
+    [Documentation]    Test Execute Shell Command kwd
+    Power On
+    Enter UEFI Shell
+    Execute UEFI Shell Command    map
```

Comment:
@JanPrusinowski Shouldn't `Execute UEFI Shell Command ` return the output? Do we need to still read it line below? This would be inconsistent with existing `Execute ... ` keywords, at least.